pkgs/tools/system/autocpu-freq/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, python3Packages, fetchFromGitHub }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "auto-cpufreq";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "AdnanHodzic";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "uVhftO6AqFnZ0uaEYRAPvVskkouNOXPtNVYXx7WJKyw=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ click distro psutil ];
+
+  # patch to prevent script copying and to disable install
+  patches = [ ./prevent-install-and-copy.patch ];
+
+  postInstall = ''
+    # copy script manually
+    cp ${src}/scripts/cpufreqctl.sh $out/bin/cpufreqctl
```

Comment:
Compared to `install()` it seems to me that the installation is incomplete (missed the font install and polkit policy). Doesn't this make this program miss some features :thinking: ?

Reply:
It seems that this script was copied from [this repo](https://github.com/konkor/cpufreq). Those files don't even exist in anywhere in auto-cpufreq, but they do exist in the other project.